### ISSUE TITLE
Wish: SWAP to be defined on vector! values

### ISSUE THREAD

**Issue by meijeru** [red/red#1961]
Again, is there perhaps a reason why it isn't already?

**Comment by dockimbel** [comment]
Yes, it needs to be implemented, series/swap is not designed to support vector! values.

**Comment by meijeru** [comment]
I consulted the source and it seems that series/swap is not used by any other type. There is string/swap inherited by file! and url!, and block/swap inherited by the other block/like types. Also, series/swap is not registered in the action table.
Now, given that vector!'s parent is string!, can't vector/swap be inherited from string/swap?
